tools/doc/html.js

```diff
@@ -135,10 +137,16 @@ function linkManPages(text) {
       if (BSD_ONLY_SYSCALLS.has(name)) {
         return `${beginning}<a href="https://www.freebsd.org/cgi/man.cgi` +
           `?query=${name}&sektion=${number}">${displayAs}</a>`;
-      }
+      } else if (LINUX_DIE_ONLY_SYSCALLS.has(name)) {
+        return `${beginning}<a href="https://linux.die.net/man/` +
+                `${number}/${name}">${displayAs}</a>`;
+      } else if (HAXX_ONLY_SYSCALLS.has(name)) {
+        return `${beginning}<a href="https://${name}.haxx.se/docs/manpage.html">${displayAs}</a>`;
+      } 
+      
       return `${beginning}<a href="http://man7.org/linux/man-pages/man${number}` +
         `/${name}.${number}${optionalCharacter}.html">${displayAs}</a>`;
-    });
+  });
```

Comment:
Nit: this should stay as before or at least the curly bracket should stay as it was before.